src/qhull_wrap.cpp

```diff
@@ -301,8 +301,25 @@ version(PyObject *self, PyObject *arg)
 }
 
 static PyMethodDef qhull_methods[] = {
-    {"delaunay", delaunay, METH_VARARGS, ""},
-    {"version", version, METH_NOARGS, ""},
+    {"delaunay", delaunay, METH_VARARGS,
+     "delaunay(x, y, /)\n"
+     "--\n\n"
+     "Compute a Delaunay triangulation.\n"
+     "\n"
+     "Parameters\n"
+     "----------\n"
+     "x, y : 1d arrays\n"
+     "    The coordinates of the point set, which must consist of at least \n"
```

Comment:
Trailing whitespace, sort of:
```suggestion
     "    The coordinates of the point set, which must consist of at least\n"
```